LICENSE

```diff
@@ -2,11 +2,11 @@ MIT License
 
 Copyright (c) 2016 Software Engineering Education - FOSS Resources
 
-Permission is hereby granted, free of charge, to any student obtaining a copy
+Permission is hereby granted, free of charge, to any person obtaining a copy
```

Comment:
You need to update to student for this file.

Reply:
Ah that one is LICENSE, should stay as person